BobbyCharms/travel-tracker
# 69 marker pop ups

## Purpose

Fixes # Adds the pop up feature to the markers on the map

This pull request will add the markers with a pop up feature. The pop ups will be displayed above the marker providing the city name with the country and state names.

## CSS and JS file changes 

1. The JS file had some code reworked. The markers are now added when the getCity function is called. This was done to fix a asynch issue. 
2. The markers will also get the pop up feature for when they are rebuild from the local storage, this part is completed in the 'initializations' section. 
3. CSS had some styles added for the pop ups


## Steps to Reproduce or Test

Place a marker on the map and then free click on the marker, a pop up should be displayed above the marker. Refresh the page and click on any existing markers, a pop up will be displayed above the marker. 

## Screenshot

![image](https://user-images.githubusercontent.com/37091234/221315743-0946dbec-538e-404b-bbeb-1f05a9fcbb6f.png)
